IBM-GTS-Workflows/PA-zOSMF-Workflows-ZFS/IBM-MF-AUTO-ZFS-DEFINE-AMS.xml

```diff
@@ -87,6 +87,42 @@
             <regularExpression>^VOLUMES\(\w*?\)</regularExpression>
         </string>
     </variable>
+    <variable name="ams-host" scope="instance" visibility="private">
+        <label>Host</label>
+        <abstract>z/OSMF host</abstract>
+        <description>Target z/OSMF Host</description>
+        <category>z/OSMF</category>
+        <string valueMustBeChoice="false" multiLine="false">
+            <minLength>1</minLength>
+            <maxLength>255</maxLength>
+        </string>
+    </variable>
+    <variable name="ams-port" scope="instance" visibility="private">
+        <label>Port</label>
+        <abstract>z/OSMF port</abstract>
+        <description>Target z/OSMF Port number</description>
+        <category>z/OSMF</category>
+        <decimal decimalPlaces="1"/>
```

Comment:
Why use a decimal for a port instead of an integer?

Reply:
Thank , I will change it.